packages/javascript/jupyterlab-plotly/package.json

```diff
@@ -31,7 +31,7 @@
     "typescript": "~3.1.1"
   },
   "dependencies": {
-    "plotly.js": "^1.51.2",
+    "plotly.js": "^1.52.1",
```

Comment:
Did you intend to downgrade plotly.js here?

Reply:
That’s an upgrade actually ;)